packages/devextreme/js/__internal/ui/html_editor/ui/aiDialog.ts

```diff
@@ -54,14 +72,18 @@ export default class AiDialog extends BaseDialog<AiDialogResult> {
 
   private _resultText = '';
 
-  private _prompt?: AICustomCommand['prompt'];
+  private _promptText = '';
```

Comment:
let's rename it somehow to show that it's related to askAI command?
`prompt` is also used for custom command